src/test/bin/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ target_link_libraries(msquictest inc gtest logging base_link)
 
 # At least /W3 must be used on all windows builds to pass compliance
 if(MSVC)
-    target_compile_options(msquictest PRIVATE /W3 /bigobj)
+    target_compile_options(msquictest PRIVATE /W3 /bigobj /fsanitize=address)
```

Comment:
I didn't realize tests weren't using the same flags. FYI, there are 3 tests that might need this flag in total.

Reply:
Build still failed.